### PR TITLE
S1t display

### PR DESCRIPTION
**Pull request by PaulREnglish**
I forgot to make a new branch for theses changes, oh well.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -31,20 +31,18 @@ main > section {
...
+    margin-top: 100px;
 }
 
 img {
     width: 100%;
 }
 
-
-
 @media only screen and (max-width: 1500px) {
```
Comment (SteeleSR) [inline styles.css:46]: Perhaps add another media query for different sizes - the title occupies two lines from 1200pixels.

Comment (SteeleSR) [reply styles.css:46]: and again at 530pixels

File: styles.css [hunk 2]
```diff
@@ -31,20 +31,18 @@ main > section {
     margin: 0 auto;
 }
 
-section h3 {
-    margin-top: 100px;
+.imageContainer {
+    width: 750px;
```
Comment (SteeleSR) [inline styles.css:35]: could consider making this a percentage so more responsive - but may distort images

Comment (SteeleSR) [reply styles.css:35]: would also allow you to get two in a row when the screen is large, and reduce to one with a media query at smaller sizes

File: styles.css [hunk 3]
```diff
@@ -55,12 +53,17 @@ img {
     * {
         font-size: 1rem;
     }
+
+    .contents {
+        width: 400px;
+    }
+
     main > section {
         width: 400px;
```
Comment (SteeleSR) [inline styles.css:62]: similarly - perhaps a percentage would be better

File: styles.css [hunk 4]
```diff
@@ -71,20 +74,18 @@ img {
...
     }
 
-
-    .imageContainer {
-        width: 100px;
+    main > section {
+        width: 200px;
     }
 
     .contentsTitle h2 {
```
Comment (SteeleSR) [inline styles.css:85]: I can't find a class called contents-title?

File: styles.css (hunk 1 above)
Comment (Zedonk1995) [reply styles.css:46]: done

File: styles.css (hunk 4 above)
Comment (Zedonk1995) [reply styles.css:85]: deleted

File: styles.css (hunk 2 above)
Comment (Zedonk1995) [reply styles.css:35]: I made the percentage change like you asked, it looks a lot better now thanks

Comment (Zedonk1995) [reply styles.css:35]: I would like to get two in a row but I don't think there's enough time and I was also don't think it's necessary to meet the requirements.

File: styles.css (hunk 3 above)
Comment (Zedonk1995) [reply styles.css:62]: done

**Pull request merged**